metron-platform/metron-api/src/main/java/org/apache/metron/pcapservice/PcapReceiverImplRestEasy.java

```diff
@@ -97,6 +97,66 @@ private static boolean isValidPort(String port) {
     }
     return false;
   }
+	  /*
+	   * (non-Javadoc)
+	   *
+	   * @see
+	   * com.cisco.opensoc.hbase.client.IPcapReceiver#getPcapsByIdentifiers(java.lang
```

Comment:
Looks like older leftover auto-generated javadoc - I'll revise the packaging and make a proper javadoc for the methods in that class.